example_test.go

```diff
@@ -24,6 +24,24 @@ func example_connect() *tarantool.Connection {
 	return conn
 }
 
+// Example demonstrates how to use SSL transport.
+func ExampleSslOpts() {
+	var opts = tarantool.Opts{
+		User:      "test",
+		Pass:      "test",
+		Transport: "ssl",
+		Ssl: tarantool.SslOpts{
+			KeyFile:  "ssl_certs/localhost.key",
+			CertFile: "ssl_certs/localhost.crt",
+			CaFile:   "ssl_certs/ca.crt",
+		},
+	}
+	_, err := tarantool.Connect("127.0.0.1:3013", opts)
+	if err != nil {
+		panic("Connection is not established")
```

Comment:
It would be nice to print `err` here too.

Reply:
Thank you, added.